OpenRA.Mods.Common/Scripting/Properties/PlayerExperienceProperties.cs

```diff
@@ -0,0 +1,48 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using Eluant;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Scripting;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Scripting
+{
+	[ScriptPropertyGroup("Player")]
+	public class PlayerExperienceProperties : ScriptPlayerProperties, Requires<PlayerExperienceInfo>
+	{
+		readonly PlayerExperience exp;
+
+		public PlayerExperienceProperties(ScriptContext context, Player player)
+			: base(context, player)
+		{
+			exp = player.PlayerActor.Trait<PlayerExperience>();
+		}
+
+		public int Experience
+		{
+			get
+			{
+				return exp.Experience;
+			}
+
+			set
+			{
+				exp.GiveExperience(exp.Experience - value);
+			}
+		}
+
+		public void GiveExperience(int num)
+		{
+			exp.GiveExperience(num);
```

Comment:
We can always add it back later if we need it.  It would be much harder to remove, so i'm in favour of being conservative.